test/test_grids.jl

```diff
@@ -585,7 +585,7 @@ end
 
             # Testing show function
             Nz = 20
-            grid = VerticallyStretchedRectilinearGrid(size=(1, 1, Nz), x=(0, 1), y=(0, 1), z_faces=collect(0:Nz).^2)
+            grid = VerticallyStretchedRectilinearGrid(architecture=arch, size=(1, 1, Nz), x=(0, 1), y=(0, 1), z_faces=collect(0:Nz).^2)
```

Comment:
@glwagner, @ali-ramadhan: does this really test `show()`?

Reply:
Right. A more explicit way to write the test is something like

```
@test try
    show(grid)
    true
catch err
    # figure out how to print the error message...
    false
end
```

Would even be nice to have a macro that writes this out like `@test_works` or something.

On the other hand just running code works fine for isolating errors in CI and seems to be practical.